Management/src/main/java/com/stack/jspdemo/model/Product.java

```diff
@@ -0,0 +1,64 @@
+package com.stack.jspdemo.model;
+
+import javax.validation.constraints.Size;
+
+import org.springframework.stereotype.Component;
+
+@Component
+public class Product {
+	private Integer id;
+	@Size(min = 10, message = "Enter at least 10 characters")
```

Comment:
Why 10 chars as the minimum?

Reply:
Change to 45 to align with my DB definition

Reply:
done